site/content/entry/citra-progress-report-2019-q1.md

```diff
@@ -0,0 +1,269 @@
+
+* User Pinging - You can now ping other users by `@ + Username or Nickname`.
+Pinged users will get a notification on their desktop and the message will be highlighted.
+
+### [Tabbed Configuration Window](https://github.com/citra-emu/citra/pull/4187) by [spycrab](https://github.com/spycrab)
+
+With many new features being added to Citra regularly, the old configuration window was getting crammed.
+While a vision for a better UI (user interface) existed since the early years of Citra, no actual work was done towards it.
+Upon [jroweboy's](https://github.com/jroweboy) request [spycrab](https://github.com/spycrab), a well known developer from dolphin-emu, undertook the implementation of such an interface.
+We plan to improve upon this interface design, to ensure best possible user experience in the future.
+
+### Open Source System Archives ([#4256](https://github.com/citra-emu/citra/pull/4256), [#4678](https://github.com/citra-emu/citra/pull/4678)) by [wwylele](https://github.com/wwylele)
+
+As part of our continued efforts towards open source system archives, wwylele implemented open source replacements for the [country list archive](https://github.com/citra-emu/citra/pull/4256) and the [mii data archive](https://github.com/citra-emu/citra/pull/4678).
+Like the name implies, the country list archive is a list of all countries in which the Nintendo eShop provides services.
+The mii data archive is a placeholder with empty models and textures, which makes mii display a sign indicating that the mii data is missing, but no longer crashes games.
+
+{{< figure src="/images/entry/citra-progress-report-2019-q1/mii_data.png"
+    title="Open Source Mii Data" >}}
+
+With these, we now have open source implementations of all the necessary system archives except the shared fonts for CHN/KOR/TWN regions.
+
+### Encrypted ROM Support ([#4181](https://github.com/citra-emu/citra/pull/4181), [#4335](https://github.com/citra-emu/citra/pull/4335), and [#4348](https://github.com/citra-emu/citra/pull/4348))
+
+For a long while now, Citra only supported decrypted ROM dumps but not encrypted ones.
+Normally, the ROM dumping process would make use of secure keys in the 3DS system and generate decrypted ROMs as output.
+But supporting encrypted ROMs meant that we had to first extract these keys from the 3DS and then make Citra use them to read and decrypt encrypted ROMS on-the-fly.
+
+Before we even got to implementing stuff on Citra's end, we had a major obstacle.
+Until now, not much research had been done to figure out which keys were needed and how to obtain these keys all together.
+Developers [wwylele](https://github.com/wwylele) and [B3N30](https://github.com/B3N30), did the necessary research and slowly implemented support for encrypted ROMs.
+Users can now follow the guide [here](https://github.com/citra-emu/citra/wiki/AES-Keys) to obtain these encryption keys, upon which Citra will be able to read encrypted ROMs or install encrypted CIAs directly.
```

Comment:
it's probably worthwhile to mention that dumping as decrypted is still recommended for general use